sdk/core/core-rest-pipeline/tsconfig.json

```diff
@@ -1,13 +1,14 @@
 {
   "extends": "../../../tsconfig.package",
   "compilerOptions": {
-    "outDir": "./dist-esm",
-    "declarationDir": "./types/latest",
     "paths": {
       "@azure/core-rest-pipeline": ["./src/index"],
     },
     "lib": ["dom", "dom.iterable"],
-    "verbatimModuleSyntax": true,
```

Comment:
please leave this enabled, the goal is to eventually make this the default for all packages

Reply:
We cannot leave this enabled as per `tshy`

```
writing tsconfig files...
verbatimModuleSyntax detected
verbatimModuleSyntax is incompatible with multi-dialect builds. Either remove
this field from tsconfig.json, or set a single dialect in the "dialects"
field in package.json, for example:

{
  "tshy": {
    "dialects": ["esm"]
  }
}
```